spec/PlayerSpec.js

```diff
@@ -0,0 +1,67 @@
+describe("Player", function() {
+  var playerOne;
+
+  beforeEach(function() {
+     playerOne = new Player("steve");
+  })
+  it(" should be able to store the player name", function() {
```

Comment:
again, is this test doing necessarily doing what it says?

Reply:
I guess I'm actually testing it can tell me the name of the player more than being able to just store the player